mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -18,6 +18,8 @@ pipeline {
                 checkout scm
                 sh '''
                     mkdir -p build && cd build
+                    # Set the path to llvm-symbolizer
+                    PATH=$PATH:/opt/rocm-4.0.0/llvm/bin/
```

Comment:
@asleepzzz Why do we need this? Why does this has to be done before the build happens? How is `llvm-symbolizer` used in our build process and how does it fail the build process if this is not done?